### PR TITLE
Fix BOM Report Calculated

### PR DESCRIPTION
**Pull request by kickapoo**
As askd to PR against staging-fixes branch

https://github.com/frappe/erpnext/pull/16042

### CODE REVIEW

File: erpnext/manufacturing/report/bom_stock_calculated/bom_stock_calculated.js [hunk 1]
```diff
@@ -11,14 +11,14 @@ frappe.query_reports["BOM Stock Calculated"] = {
...
 		},
-        	{
-	            "fieldname": "qty_to_make",
-        	    "label": __("Quantity to Make"),
-        	    "fieldtype": "Int",
-        	    "default": "1"
-	       },
-
-		 {
+		{
```
Comment (rmehta) [inline erpnext/manufacturing/report/bom_stock_calculated/bom_stock_calculated.js]: @kickapoo why is the whitespace messed up??  this is really wasting my review time. So many times we have discussed this. @dottenbr

**Review by rmehta — CHANGES_REQUESTED** [review]

File: erpnext/manufacturing/report/bom_stock_calculated/bom_stock_calculated.js (hunk 1 above)
Comment (kickapoo) [reply erpnext/manufacturing/report/bom_stock_calculated/bom_stock_calculated.js]: @rmehta I fix the previous commiter indentation issue to follow tabs etc.
Sorry for wasting your time not indention doing so.

**Comment by kickapoo** [comment]
Its one line so frappe team can fix it if they want to. 

@rmehta Sorry for wasting your time, no intention of doing it.

**Comment by dottenbr** [comment]
@kickapoo please fix your PR

**Pull request closed without merging**